UserSettings/Layouts/CurrentMaximizeLayout.dwlt

```diff
@@ -0,0 +1,927 @@
+%YAML 1.1
```

Comment:
User setting files are not tracked anymore with a `.gitignore` update that occurred after you added this file. You can delete this file and then push that deletion to remove it. (It will get generated again by the Unity Editor.) (Here is how you can remove the file from the repository without deleting it locally: https://stackoverflow.com/a/1143800.)
- [ ] Remove this file

Reply:
It looks correct to me. I don't see this file on your branch anymore.